docs/en/engines/database-engines/materialize-mysql.md

```diff
@@ -0,0 +1,160 @@
+---
+toc_priority: 29
+toc_title: MaterializeMySQL
+---
+
+# MaterializeMySQL {#materialize-mysql}
+
+ Creates the tables, existing in MySQL, in the ClickHouse and inputs the data at the same time. 
+
+ ClickHouse server works as MySQL replica. It reads binlog and performs DDL and DML queries.   
+
+## Creating a Database {#creating-a-database}
+
+``` sql
+CREATE DATABASE [IF NOT EXISTS] db_name [ON CLUSTER cluster]
+ENGINE = MaterializeMySQL('host:port', ['database' | database], 'user', 'password')
+```
+
+**Engine Parameters**
+
+-   `host:port` — MySQL server address.
+-   `database` — Remote database name.
+-   `user` — MySQL user.
+-   `password` — User password.
+
+## Virtual columns {#virtual-columns}
+
+ When working with the MaterializeMySQL database engine, [ReplacingMergeTree](../../engines/table-engines/mergetree-family/replacingmergetree.md) tables are used with virtual `_sign` and `_version` columns.
```

Comment:
```suggestion
 When working with the `MaterializeMySQL` database engine, [ReplacingMergeTree](../../engines/table-engines/mergetree-family/replacingmergetree.md) tables are used with virtual `_sign` and `_version` columns.
```

Reply:
Also I didn't understand the meaning of the phrase. Maybe, "In the MaterializeMySQL-engine database all the tables have the ReplacingMergeTree engine ..."